4.4/Dockerfile

```diff
@@ -8,8 +8,13 @@ ENV LANG=C.UTF-8
 
 WORKDIR /app
 
-RUN \
-    --mount=type=cache,target=/var/cache/apt \
+# Inspired by https://github.com/reproducible-containers/buildkit-cache-dance?tab=readme-ov-file#apt-get-github-actions
+RUN  \
+    --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Comment:
I'm a bit confused why concurrent access was even possible. I was under the impression that these are two totally different machines/vms and don't share anything, let alone disk. Where is this cache being saved to?